engines/nancy/detection.cpp

```diff
@@ -88,12 +88,7 @@ static const Nancy::NancyGameDescription gameDescriptions[] = {
 	{ // MD5 by fracturehill
 		{
 			"nancy1", nullptr,
-			{
-				{ "data1.hdr", 0, "39b33ad649d3e7261508d3c6907f237f", 139814 },
-				{ "data1.cab", 0, "f900861c47b0cb88191f5c6189db6cb1", 1916153 },
-				{ "data2.cab", 0, "9c652edb9846a721839cb7e1dcc94a3e", 462008320 },
-				AD_LISTEND
-			},
+			AD_ENTRY1s("is:data1.cab:ciftree.dat", "A:e1cd21841ab1b83a0ea0755ce0254cbc", 4480956),
```

Comment:
Initially, I was thinking about something like
```C++
AD_ENTRY1s("data1.cab", "A:is:ciftree.dat:e1cd21841ab1b83a0ea0755ce0254cbc", 4480956)
```

But I like your schema better

Reply:
Thank you! I was initially planning to do that but quickly realized it makes more sense to keep information about the archive outside of the md5 sum.